pom.xml

```diff
@@ -517,7 +517,7 @@
 			<dependency>
 				<groupId>net.sf.ehcache</groupId>
 				<artifactId>ehcache</artifactId>
-				<version>2.10.10.1.14</version>
```

Comment:
Is this change required by the ticket?

Reply:
I added that change because the maven build would show errors due to the new version of ehcache which wasn't working on my local machine